coldfront/core/project/views.py

```diff
@@ -4162,13 +4162,15 @@ def form_valid(self, form):
                 allocation_obj = Allocation.objects.get(project=project_obj)
                 allocation_user = \
                     allocation_obj.allocationuser_set.get(user=removed_user.user)
-                allocation_denied_status, _ = \
```

Comment:
I hadn't noticed that there is actually an `AllocationUserStatusChoice` called "Removed"; it seems appropriate to set that here as well.

That would also simplify this [block](https://github.com/ucb-rit/coldfront/pull/288/files#diff-5704bb865d9a2751bf4d43c1f88d4ce8d2bd8c175513f49a60d682e5bc33b67aR214); you could just filter for `AllocationUsers` with that status.